server/app.py

```diff
@@ -1,10 +1,24 @@
 from flask import Flask
 from api.ping_handler import ping_handler
 from api.home_handler import home_handler
-
+from flask_sqlalchemy import SQLAlchemy
 
 app = Flask(__name__)
 
+#creates a postgres database connection with postgresql://username:password@localhost/dbname
+app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://postgres:1234@localhost/team-winter'
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+
+db = SQLAlchemy(app)
+
+#test database, remove as necessary.
+class Test(db.Model):
+    __tablename__ = 'Tests'
+    id = db.Column(db.Integer, primary_key = True)
```

Comment:
This should be in a separate folder structure, all models should have their own folder structure

Reply:
Yes, I was just testing it real quick!